### PR TITLE
Update wcoss2 modulefile

### LINKED ISSUE #20: Update WCOSS2 Modules
The UFS now points to the hpc-stack suite installed in /apps/test/hpc-stack/i-19.1.3.304__m-8.1.12__h-1.14.0__n-4.9.2__p-2.5.10__e-8.4.2.  The GSI-utils modules should be updated to point to the same stack.

### PR DESCRIPTION
**Pull request by RussTreadon-NOAA**
This PR removes `/apps/ops/para/libs` from `gsiutils_wcoss2.intel.lua`.  Additionally module versions for w3emc, nemsio, and ncdiag are updated to be consistent with GSI [gsi_wcoss2.intel.lua](https://github.com/NOAA-EMC/GSI/blob/develop/modulefiles/gsi_wcoss2.intel.lua).

Resolves #20

### CODE REVIEW

**Comment by RussTreadon-NOAA** [comment]
@DavidHuber-NOAA , automated tests for gsi-monitor failed.  Any idea why gsi-monitor being tested in gsi-utils?  [GSI-Monitor](https://github.com/NOAA-EMC/GSI-Monitor) has its own repository.

FYI, I successfully built `feature/wcoss2` on Cactus.

**Comment by DavidHuber-NOAA** [comment]
@RussTreadon-NOAA That was a copy-paste error when updating the CI for #19.  It is in fact only trying to build gsi-utils but it is mislabeled.

The cause of the failure is that `mpif90` is missing.  Since this PR only affect Intel compilers on WCOSS2, I'm OK with a GCC failure.  I will investigate/fix the CI issues in a separate PR today.

**Review by DavidHuber-NOAA — APPROVED** [review]
Changes look good.  Would you be able to run this through a cycled test on Cactus?

**Comment by RussTreadon-NOAA** [comment]
Corrected `gsi-monitor` typo at [6d39a55](https://github.com/NOAA-EMC/GSI-utils/pull/44/commits/6d39a557722119d55b02b637229c05b968419c15).  Attempt to fix gcc compilation error at [6d39a55](https://github.com/NOAA-EMC/GSI-utils/pull/44/commits/6d39a557722119d55b02b637229c05b968419c15) failed.

Installing `feature/wcoss2` on Cactus and will run GSI-base DA CI (C96C48_hybatmDA).

**Comment by RussTreadon-NOAA** [comment]
GSI-based (C96C48_hybatmDA) and JEDI-based (C96C48_ufs_hybatmDA) CI run on Cactus by updating `sorc/gsi_utils.fd` at 6d39a55 in working copy of `RussTreadon-NOAA:feature/wcoss2_ufsda`.  All jobs from both CI tests ran to completion

GSI-based (C96C48_hybatmDA)
```
russ.treadon@clogin05:/lfs/h2/emc/ptmp/russ.treadon/EXPDIR/prtest_gsi> rocotostat -d prtest_gsi.db -w prtest_gsi.xml -c all -s                
   CYCLE         STATE           ACTIVATED              DEACTIVATED     
202112201800        Done    May 29 2024 13:46:37    May 29 2024 14:25:23
202112210000        Done    May 29 2024 13:46:37    May 29 2024 16:52:10
202112210600        Done    May 29 2024 13:46:37    May 29 2024 16:55:14
```

JEDI-based (C96C48_ufs_hybatmDA)
```
russ.treadon@clogin05:/lfs/h2/emc/ptmp/russ.treadon/EXPDIR/prtest> rocotostat -d prtest.db -w prtest.xml  -c all -s
   CYCLE         STATE           ACTIVATED              DEACTIVATED     
202402231800        Done    May 29 2024 13:46:47    May 29 2024 14:20:14
202402240000        Done    May 29 2024 13:46:47    May 29 2024 16:55:11
```

**Comment by RussTreadon-NOAA** [comment]
GSI-utils PR #45 corrects automated CI.  Revert [`6d39a55`](https://github.com/NOAA-EMC/GSI-utils/pull/44/commits/6d39a557722119d55b02b637229c05b968419c15) in `feature/wcoss2`.  Done at 0005fc9.

**Comment by RussTreadon-NOAA** [comment]
@DavidHuber-NOAA :  GSI-based (C96C48_hybatmDA) and JEDI-based (C96C48_ufs_hybatmDA) successfully run on Cactus.  

Spot check output from this run with CI tests reported in g-w PR [#2620](https://github.com/NOAA-EMC/global-workflow/pull/2620). The PR [#2620](https://github.com/NOAA-EMC/global-workflow/pull/2620) CI runs used [gsi_utils.fd @ bb03e17](https://github.com/NOAA-EMC/GSI-Utils/tree/bb03e172e0d0d9c56d6da7788ca033bfb5ef5119).  The CI tests reported in this PR use `gsi_utils.fd` at [6d39a55](https://github.com/NOAA-EMC/GSI-utils/commit/6d39a557722119d55b02b637229c05b968419c15).

Are there any more checks or tests you would like me to make or run?

**Comment by DavidHuber-NOAA** [comment]
Nope, I think this is good to go.  Thanks @RussTreadon-NOAA!

**Comment by RussTreadon-NOAA** [comment]
> Nope, I think this is good to go. Thanks @RussTreadon-NOAA!

Thank you @DavidHuber-NOAA .  Would you re-review and approve?  Seems my commits wiped out your earlier approval.

**Review by DavidHuber-NOAA — APPROVED** [review]

**Comment by RussTreadon-NOAA** [comment]
@DavidHuber-NOAA :  Looks like your approval is there but I still see

> Review required
At least 1 approving review is required by reviewers with write access. 

Maybe your approval isn't sufficient because you do not have write access to this repo.  Is this possible?

**Comment by DavidHuber-NOAA** [comment]
@RussTreadon-NOAA I think you are correct.  Perhaps @aerorahul can approve.

File: modulefiles/gsiutils_wcoss2.intel.lua [hunk 1]
```diff
@@ -40,15 +42,8 @@ load(pathJoin("sigio", sigio_ver))
...
 load(pathJoin("crtm", crtm_ver))
-
-load(pathJoin("prod_util", prod_util_ver))
-
-pushenv("HPC_OPT", "/apps/ops/para/libs")
-prepend_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/compiler/intel/19.1.3.304")
-prepend_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/mpi/intel/19.1.3.304/cray-mpich/8.1.7")
-
-load("ncio/1.1.2")
-load("ncdiag/1.0.0")
```
Comment (aerorahul) [inline modulefiles/gsiutils_wcoss2.intel.lua:52]: nice!

**Review by aerorahul — APPROVED** [review]
lgtm

**Comment by RussTreadon-NOAA** [comment]
Thank you @aerorahul

**Pull request merged**